simde/x86/sse4.1.h

```diff
@@ -1902,11 +1902,17 @@ simde_mm_test_mix_ones_zeros (simde__m128i a, simde__m128i mask) {
     a_ = simde__m128i_to_private(a),
     mask_ = simde__m128i_to_private(mask);
 
-  for (size_t i = 0 ; i < (sizeof(a_.u64) / sizeof(a_.u64[0])) ; i++)
-    if (((a_.u64[i] & mask_.u64[i]) != 0) && ((~a_.u64[i] & mask_.u64[i]) != 0))
-      return 1;
+  #if defined(SIMDE_ARM_NEON_A32V7_NATIVE)
+    int64x2_t s640 = vandq_s64(a_.neon_i64, mask_.neon_i64);
+    int64x2_t s641 = vandq_s64(~a_.neon_i64, mask_.neon_i64);
```

Comment:
`~a_.neon_i64` is actually [a GCC extension](https://gcc.gnu.org/onlinedocs/gcc/Vector-Extensions.html); with normal NEON you can't use the NOT operator on a vector.

What you want here is `vmvnq_s64`, but of course that doesn't exist (which is surprising, NEON is normally really good about that type of thing).  Luckily since this is just a simple bitwise operation the types don't really matter, so you can do something like

```c
int64x2_t s641 = vandq_s64(vreinterpretq_s64_s32(vmvnq_s32(vreinterpretq_s32_s664(a_.neon_i64))), mask_.neon_i64)
```

It's a bit ugly, but it works :)

Reply:
I see, I will fix this.
Thank You

Reply:
fixed this in #533